boefjes/boefjes/local.py

```diff
@@ -50,6 +50,9 @@ def run(self, boefje_meta: BoefjeMeta, environment: dict[str, str]) -> list[tupl
         boefjes = self.local_repository.resolve_boefjes()
         boefje_resource = boefjes[boefje_meta.boefje.id]
 
+        if not boefje_resource.module and boefje_resource.boefje.oci_image:
```

Comment:
Nullable fields should probably be checked explicitly using `is None`. For `oci_image` it may be different, if the intention is to also validate against an empty string.